src/plugins/dashboard/public/services/dashboard_content_management/types.ts

```diff
@@ -39,7 +39,7 @@ export interface DashboardContentManagementRequiredServices {
 
 export interface DashboardContentManagementService {
   findDashboards: FindDashboardsService;
-  deleteDashboards: (ids: string[]) => void;
+  deleteDashboards: (ids: string[]) => Promise<void>;
```

Comment:
Unrelated, but noticed during my investigation: this method was `async` but, since the return value type didn't match, you would get a warning when `awaiting` it.